scripts/cli/util/distributePOLYX.js

```diff
@@ -0,0 +1,59 @@
+// Set options as a parameter, environment variable, or rc file.
+require = require("esm")(module /*, options*/);
+module.exports = require("./init.js");
+
+let { reqImports } = require("./init.js");
+const minimist = require('minimist');
+const args = minimist(process.argv.slice(2), {
+    string: 'account'
+});
+
+async function main() {
+    const api = await reqImports.createApi();  
+    const account = await reqImports.generateEntityFromUri(api, args.account);  
+    const amount = args.amount; 
+
+    let empty_did = "0x0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
const?

Reply:
thanks for the advise, changed empty_did to a const.